File: server/router/yunguan/product/productConfig.go
package product

import (
	"github.com/gin-gonic/gin"
	v1 "ygpt/server/api/v1"
)

type ProductConfigRouter struct {
}

func (s *ProductConfigRouter) InitProductConfigRouter(Router *gin.RouterGroup) {
	dataRouter := Router.Group("product/config")
	supplyInfoApi := v1.ApiGroupApp.YunGuanApiGroup.ProductConfigApi

	{
		dataRouter.POST("addConfigInfo", supplyInfoApi.AddConfigInfo)           // 添加数据
		dataRouter.PUT("updateConfigInfo", supplyInfoApi.UpdateConfigInfo)      // 修改数据
		dataRouter.DELETE("deleteConfigInfo", supplyInfoApi.DeleteConfigInfo)   // 删除数据
		dataRouter.POST("queryConfigInfo", supplyInfoApi.QueryConfigInfo)       // 获取数据列表
		dataRouter.POST("queryAllConfigInfo", supplyInfoApi.QueryAllConfigInfo) // 获取数据列表
	}
}

File: web/src/api/yunguan/product/productConfig.js
import request from "@/utils/request";

// 添加系统运行工具
export function addConfigInfo(modelData) {
    return request({
        url: "product/config/addConfigInfo",
        method: "post",
        data: modelData,
    });
}

// 修改系统运行工具
export function updateConfigInfo(data) {
    return request({
        url: "product/config/updateConfigInfo",
        method: "put",
        data: data,
    });
}

// 获取系统运行工具列表
export function getConfigInfoList(params) {
    return request({
        url: "product/config/queryConfigInfo",
        method: "post",
        data: params,
    });
}
// 获取系统运行工具列表
export function getAllConfigInfoList() {
    return request({
        url: "product/config/queryAllConfigInfo",
        method: "post",
    });
}

// 删除系统运行工具
export function deleteConfigInfo(ID) {
    return request({
        url: "product/config/deleteConfigInfo",
        method: "delete",
        data: { ID: ID },
    });
}
